README.md

```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
 
 The projects use [npm semantic versioning](https://docs.npmjs.com/about-semantic-versioning)
 
+## Project boards
+
+Between the 5th of June and 17th of July, there is a sprint board used for organising the backlog for each two weeks sprint cycle.
+
+- [BBC News Labs - Digital Paper Edit - Sprint Board](https://github.com/orgs/bbc/projects/33) is a github project board [with linked repository](https://help.github.com/en/articles/linking-a-repository-to-a-project-board) where issues are added from the relevant repository listed in architecture section.
```

Comment:
Again super picky but you might want to do the capitals for GitHub :) Up to you. It's clear either way.